src/WebtoolsGeocoding.php

```diff
@@ -134,4 +151,17 @@ protected function getAddressData(\stdClass $feature): array {
 
     return $address_data;
   }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getRequest(string $url): RequestInterface {
+    $request = $this->getMessageFactory()->createRequest('GET', $url);
```

Comment:
Instead of copying the line from the `parent` why don't call the parent's method?

```php
$request = parent::getRequest($url);
```

Reply:
Ah yes, good catch. I had an earlier iteration which changed this line to pass a `$headers` array but I changed it back to the original line. This is better indeed to call the parent method.